src/PhpSpreadsheet/Writer/Xls/Xf.php

```diff
@@ -498,14 +504,15 @@ private static function mapTextRotation($textRotation)
      */
     private static function mapLocked($locked)
     {
-        switch ($locked) {
-            case Protection::PROTECTION_UNPROTECTED:
-                return 0;
-        }
-
-        return 1;
+        return array_key_exists($locked, self::LOCK_ARRAY) ? LOCK_ARRAY[$locked] : 1;
```

Comment:
I'm not sure what happened here. Commit dc4f568 had the error you are asking to correct, but I replaced it immediately with commit dc908f5, which has the correction. There would have been a very short window indeed for you to see the wrong code. If you look at lines 507 and 525, they have the appropriate change. So I am marking this as resolved without making any further changes. If I am misinterpreting something, please let me know.

Reply:
Actually, I decided to change a comment just in case. Commit 9c279fb. If the code still looks wrong to you, I will cancel this PR and open another in its place.

Reply:
Actually, I decided to add a comment just in case. Commit 9c279fb. If the code still looks wrong to you, I will cancel this PR and open another in its place.